Tests/ComposableArchitectureTests/EffectCancellationTests.swift

```diff
@@ -4,15 +4,15 @@ import XCTest
 @testable import ComposableArchitecture
 
 final class EffectCancellationTests: XCTestCase {
+  struct CancelToken: Hashable {}
```

Comment:
Ah good change. Local cancel tokens are nice for actual reducers, but not for tests!